Core/CwCheat.cpp

```diff
@@ -263,7 +269,10 @@ inline void trim2(std::string& str) {
 	else str.erase(str.begin(), str.end());
 }
 
-std::vector<std::string> CWCheatEngine::GetCodesList() { //Reads the entire cheat list from the appropriate .ini.
+std::vector<std::string> CWCheatEngine::GetCodesList(std::string file) { //Reads the entire cheat list from the appropriate .ini.
+	if (file.empty()) {
+		file = activeCheatFile;
```

Comment:
Well, I suppose this isn't doing anything since `activeCheatFile` is being set otherwise.

-[Unknown]